reforger/index.html

```diff
@@ -0,0 +1,100 @@
+---
+title: Arma Reforger | Swedish Forces Pack
+images:
+  -
+    path: images/ak4or.png
+    title: Marksman
+  -
+    path: images/ksp58.png
+    title: Machinegunner
+  -
+    path: images/leaders.png
+    title: Squadleaders
+  -
+    path: images/pskott.png
+    title: Antitank soldier
+
+---
+
+<div class="container">
+  <h1>Swedish Forces Pack</h1>
+</div>
+
+<div class="container">
+  <h2>Description</h2>
+
+  <p>
+   The original Swedish Armed Forces experience for ArmA are now entering the next generation with the first release for Arma Reforger.
```

Comment:
Fix indentation. It's Arma, not ArmA
```suggestion
    The original Swedish Armed Forces experience for Arma are now entering the next generation with the first release for Arma Reforger.
```